src/strings/PowerFxResources.en-US.resx

```diff
@@ -6615,4 +6619,8 @@
     <value>This feature is deprecated and is no longer supported.</value>
     <comment>An error message for deprecated features.</comment>
   </data>
+  <data name="WarnDeferredType" xml:space="preserve">
+    <value>Warning: Deferred type provided.</value>
+    <comment>Warning given when the input has deffered Type.</comment>
```

Comment:
nit, spelling deffered  -> deferred  . Also, add a bit of context on what "Deferred" means, so the translators can have an easier time.